lib/clevertap_plugin.dart

```diff
@@ -1135,4 +1135,14 @@ class CleverTapPlugin {
     return await _dartToNativeMethodChannel.invokeMethod(
         'setLocale', localeString);
   }
+
+  ///Fetch Client Side In-Apps
+  static Future<bool?> fetchInApps() async {
```

Comment:
I think this can be changed. But historically we have been using nullable types everywhere for some reason.
@kushCT @shivamsharma2710 Could you confirm on this?

Reply:
@CTLalit this `Future` is a placeholder for a value that hasn't been computed yet but will be available later. So  there's a possibility that the boolean value might be null and might lead to runtime exception. So to be on safe side most of the objects has been declared as `Nullable` in the file.

Reply:
@kushCT Correct. We've have been using nullable futures since either it can successfully resolve to a bool value or return null in case of any failure or when the result is not available.